### ISSUE TITLE
VoTT is deprecated

### ISSUE THREAD

**Issue by 21r8390** [dotnet/docs#28835]
Model Builder is currently still in preview, which is why the [documentation](https://docs.microsoft.com/en-us/dotnet/machine-learning/tutorials/object-detection-model-builder) is most likely not yet complete. 

However, this documentation uses the [VoTT](https://github.com/microsoft/VoTT) software, which can be downloaded from GitHub. However, the repo and this software are outdated and archived. What @wbreza [announced](https://github.com/microsoft/VoTT/commit/47f5e0073092d706a216d44688565cd8d3e18dfc).

Since Model Builder is a new product, it would be nice if outdated examples are not used before the release. 
Is there no alternative to this program that proves the same utility? 

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 63a4f3b7-a2a6-760a-534b-22788ba77b5c
* Version Independent ID: 1bee7901-164f-d9a7-9e7b-fd0ba55ad42e
* Content: [Tutorial: Detect objects in images with Model Builder - ML.NET](https://docs.microsoft.com/en-us/dotnet/machine-learning/tutorials/object-detection-model-builder)
* Content Source: [docs/machine-learning/tutorials/object-detection-model-builder.md](https://github.com/dotnet/docs/blob/main/docs/machine-learning/tutorials/object-detection-model-builder.md)
* Product: **dotnet-ml**
* GitHub Login: @luisquintanilla
* Microsoft Alias: **luquinta**

**Comment by luisquintanilla** [comment]
Hi @21r8390 

Thanks for your questions. 

> Model Builder is currently still in preview, which is why the [documentation](https://docs.microsoft.com/en-us/dotnet/machine-learning/tutorials/object-detection-model-builder) is most likely not yet complete.

Would you be able to add more details on what's missing from the documentation?

> Is there no alternative to this program that proves the same utility?

Although we are aware that VoTT is archived and no longer actively maintained, for the Model Builder use case, the existing functionality makes the process of labeling / tagging images easier while offering a diverse set of export formats. That being said, we're open to feedback to understand whether the current solution is either not solving your problems or getting in the way. Is there any specific functionality currently missing from your workflow that VoTT doesn't address?

**Comment by 21r8390** [comment]
Thank you for the reply.

> Would you be able to add more details on what's missing from the documentation?

I personally find that the documentation is complete and very well written. However, the documentation states that Model Builder is currently in preview, so I wanted to note that.

VoTT fulfills all the tasks and requirements needed for this tutorial and beyond. However, from the user / beginner point of view, it is unpleasant to be greeted with the message that the product is archived. 
It would be more pleasant if this message would not come or some hint with explanation is in the doc. This way people would know that the program still has all the functions and can be safely used.

**Comment by pjmagee** [comment]
Is there something else that is a good replacement of VoTT that is currently actively maintained and improved for such things? It feels strange to follow and use VoTT if it's archived.

**Comment by normanhh3** [comment]
@luisquintanilla I agree with the comments above.  I am just getting started with some Vision AI through the ML.NET package and also found references to VOTT.  If the tool is no longer being maintained (including security fixes as one fork is attempting to do) it will be a challenge to communicate to my corporate security team that this tool is ok to use.

Further, with each passing day, the fact that this tool is not being maintained means that critical updates just to keep it running as dependencies change won't be occurring.

This solution (https://github.com/SysCV/nutsh) looks like an excellent replacement tool from the perspective of quickly developing custom datasets, however, this tool is not yet [supported on Windows](https://github.com/SysCV/nutsh/issues/1).

It is quite clear to me that for the further development and support of custom Vision AI solutions using the Microsoft and Azure tool suite will require either endorsement and support of some other tool, or reconsideration of the non-supported situation with VOTT.

Thanks for letting us bend your ear a bit. :-)

**Comment by acrigney** [comment]
Great ear bending as above, and it seems no response for 10 months? I have been using ML.NET now on many projects and before I was doing my image detection without any transfer learning using object categorisation that worked really well, but my new use case requires very fine identification of objects in images so I think I need object detection. VoTT looks like a great tool, it seems that Microsoft has decided that this tool is too good to be just free, as has happened a lot in the past unfortunately, LightSwitch, ISpy etc, but I really love how you have embraced open source Azure Functions, but again as serverless is only still a very small part of the web still again that seems like a strategic direction. So can the managers have a rethink, or are you just planning an even better tool and my suspicions are completely off track!!
